### ISSUE TITLE
note-path is not styled with provided note default color

### ISSUE THREAD

**Issue by davenquinn** [susielu/d3-annotation#40]
The path extending the connector above/beside the note is not given a stroke style in d3-annotation v2. I am running Chomium within electron, and it is not rendered.

<img width="231" alt="screen shot 2017-09-20 at 5 17 12 am" src="https://user-images.githubusercontent.com/1537910/30643724-ed5a2240-9dc4-11e7-855d-c820706fd7ba.png">

There is no stroke style applied to the appropriate element.

<img width="465" alt="screen shot 2017-09-20 at 5 20 44 am" src="https://user-images.githubusercontent.com/1537910/30643841-48fa3fe0-9dc5-11e7-837f-16464ab5bfe2.png">

This can be rectified for now by manually applying the style after drawing the annotations, as such (pardon my coffeescript)

```coffeescript
  el.call makeAnnotations

  d3.selectAll 'g.annotations g.annotation'
    .each (d)->
      d3.select @
        .select '.note-line'
          .attr 'stroke', d.color
```

Expected output:
<img width="156" alt="screen shot 2017-09-20 at 5 34 14 am" src="https://user-images.githubusercontent.com/1537910/30643874-5fc2a5dc-9dc5-11e7-9bd0-92a820dc548d.png">

Thanks for the amazing library!

**Comment by susielu** [comment]
Thanks for finding this. I'm on vacay without a laptop for a few weeks will
take a look once I'm home.

**Comment by Lacroute** [comment]
Sorry for the duplicate https://github.com/susielu/d3-annotation/issues/42
To fix it, apply a color to the stroke css property.

**Comment by susielu** [comment]
This should be resolved in v 2.1.0. thanks for submitting this bug.
